doc/system_specific_setup.rst

```diff
@@ -1,225 +1,266 @@
+using the program pip ("PIP Installs Packages"). Besides ``pip``
+it is also recommended to install the graphviz package to be
+able to visualise dependency graphs. This is optional and the associated
+routine will silently return if the graphviz bindings are not
+installed.
 
-Get a terminal window. You can do this by pressing <ctrl><Alt><t>
-together, or click the top left "search" icon and type "terminal".
+.. _ubuntu_user:
 
-Install PSyclone
-^^^^^^^^^^^^^^^^
+Installing dependencies on Ubuntu
++++++++++++++++++++++++++++++++++
 
-Change directory to where you would like to place the code (where
-<PSYCLONEHOME> refers to where you would like to place the code):
+On Ubuntu ``pip`` and ``graphviz`` is installed using ``apt-get``. Remember
+that graphviz is optional.
 ::
 
-   > cd <PSYCLONEHOME>
-
-Now download and extract the latest release of PSyclone, e.g.:
-
-.. parsed-literal::
-
-   > wget https://github.com/stfc/PSyclone/archive/\ |release|\ .tar.gz
-   > gunzip \ |release|\ .tar.gz
-   > tar xf \ |release|\ .tar
-   > rm \ |release|\ .tar
-
-Set your python path and path appropriately:
-
-.. parsed-literal::
-
-   > cd PSyclone-\ |release|\ 
-   > export PYTHONPATH=`pwd`/src:${PYTHONPATH}
-   > export PATH=`pwd`/bin:${PATH}
+   > sudo apt-get install python-pip graphviz
 
-You may want to set these paths permanently (e.g. by editing your
-${HOME}/.bashrc file if you run the BASH shell).
+.. _opensuse_user:
 
-Install Python packages using apt package manager
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Installing dependencies on OpenSUSE
++++++++++++++++++++++++++++++++++++
 
-numpy is required to run PSyclone
+While the vanilla OpenSUSE installation includes ``pip``,
+the installed version only works for Python 3. So the
+python 2 version of PIP still needs to be installed. Note
+that the graphviz package is installed by default.
 ::
 
-   > sudo apt-get install python-numpy
+    > sudo zypper install python-pip
 
-pyparsing is required by PSyclone
-::
+.. warning::
+    PIP for python2 on OpenSUSE is called ``pip2.7``. So you need
+    to replace the ``pip`` command with ``pip2.7`` in all commands in
+    the following sections. 
 
-   > sudo apt-get install python-pyparsing
 
-.. _install_pip:
 
-Install pip
-^^^^^^^^^^^
+Installing PSyclone
+^^^^^^^^^^^^^^^^^^^
+Change your working directory to where you would like to place the code and 
+download the latest stable release of PSyclone. 
 
-The pip tool enables Python packages to be installed from the Python
-Package Index (https://packaging.python.org/installing/). Install it like so:
-::
+.. parsed-literal::
 
-   > sudo apt-get install python-pip
+   > cd <PSYCLONEHOME>
+   > wget https://github.com/stfc/PSyclone/archive/\ |release|\ .tar.gz
+   > gunzip \ |release|\ .tar.gz
+   > tar xf \ |release|\ .tar
+   > rm \ |release|\ .tar
+   > cd PSyclone-\ |release|
+   > export PYTHONPATH=`pwd`/src:${PYTHONPATH}
+   > export PATH=`pwd`/bin:${PATH}
```

Comment:
This is a comment for @rupertford really - we should use `pip install --user .` rather than setting PYTHONPATH and PATH.